components/input/index.jsx

```diff
@@ -331,6 +342,29 @@ const Input = createReactClass({
 		const hasRightIcon =
 			!!this.props.iconRight ||
 			(this.props.iconPosition === 'right' && !!this.props.iconName);
+		let fieldLevelHelpTooltip;
+		if (
+			(this.props.label ||
+				(this.props.assistiveText && this.props.assistiveText.label)) &&
+			this.props.fieldLevelHelpTooltip
+		) {
+			const defaultProps = {
+				triggerClassName: 'slds-form-element__icon',
+				children: (
+					<Button
+						assistiveText={{ label: 'Help' }}
```

Comment:
This needs to be able to be internationalized and added to `assistiveText` object. `fieldLevelHelpButton`, maybe?